File: server/app/structs/FooRoute.ts
import { Cmd } from './Cmd';
import { RouteBase } from './RouteBase';

import { proto } from "proto-structs";

export default class FooRoute extends RouteBase {
    public Foo: Cmd = Cmd.create('connector.entryHandler.onFoo', proto.Foo, proto.Bar);
    public LargeNumber: Cmd = Cmd.create('connector.entryHandler.onLargeNumber', proto.LargeNumber, proto.LargeNumber);
}

File: unity-client/Assets/Plugins/Route/FooRoute.cs
public class FooRoute : RouteBase {
    public Cmd Foo = new Cmd("connector.entryHandler.onFoo", typeof(Proto.Foo), typeof(Proto.Bar));
    public Cmd LargeNumber = new Cmd("connector.entryHandler.onLargeNumber", typeof(Proto.LargeNumber), typeof(Proto.LargeNumber));
    public Cmd NotifyLargeNumber = new Cmd("connector.entryHandler.onNotifyLargeNumber", typeof(Proto.LargeNumber), typeof(Proto.LargeNumber));
}